src/CoreSpotlight/CSCompat.cs

```diff
@@ -13,6 +14,23 @@ public CSCustomAttributeKey () : this (String.Empty)
 		{
 		}
 	}
+
+#if !TV
+	public partial class CSSearchQueryContext {
+
+#if !NET
+		[Obsolete ("This property was removed. The getter always returns null and the setter throws and InvalidOperationException.")]
+#else
+		[UnsupportedOSPlatform ("ios16.1")]
```

Comment:
The whole file has a `#if !NET && IOS`, which means this won't be included in .NET.

Reply:
I'm stupid.